IDE/STM32Cube/wolfssl_example.c

```diff
@@ -1468,25 +1471,278 @@ static void PrintMemStats(void)
 #endif
 }
 
-#if 0
-static void* wolfMallocCb(size_t size)
+#if !defined(WOLFCRYPT_ONLY) && defined(WOLFSSL_TLS13)
+/* UART DMA IO Routines */
+#ifndef B115200
+#define B115200 115200
+#endif
+
+/* Max buffer for a single TLS frame */
+#ifndef MAX_RECORD_SIZE
+#define MAX_RECORD_SIZE (16 * 1024)
+#endif
+
+typedef struct {
+    int curr_index;
+    int data_len;
+    char buf[MAX_RECORD_SIZE];
```

Comment:
Depending on `HAVE_MAX_FRAGMENT` this could be reduced. Suggest making this an overridable build time macro where if not set defaults to `MAX_RECORD_SIZE`. If you haven't seen max fragment.. check the run-time options `WOLFSSL_MFL_MIN` -> `WOLFSSL_MFL_MAX` in ssl.h.